README.rst

```diff
@@ -0,0 +1,115 @@
+PyCraft
+=======
+
+A community driven fork of `foglemans "Minecraft"
+repo <https://github.com/fogleman/Minecraft>`_.
+
+|#pycraft on freenode|
+
+
+Motivation
+----------
+
+The original project had great ideas and implemented really cool things,
+but it was primarily designed around teaching.  This project is intended
+to create a community-driven engine for complicated voxel/roguelike
+games which ultimately can use and extend
+`GURPS <https://en.wikipedia.org/wiki/GURPS>`__
+
+    The Generic Universal RolePlaying System, or GURPS, is a tabletop
+    role-playing game system designed to allow for play in any game
+    setting.
+
+This project will be very permissive in accepting pull-requests.
+
+
+Screenshot
+----------
+
+.. figure:: screenshot.png
+   :alt:
+
+Virtual Environment (Recommended)
+---------------------------------
+
+.. code:: bash
+
+    # create a virtual environment
+    virtualenv -p python3 ~/.venv/pycraft # (or wherever)
+    # you may need to add execute permissions
+    chmod -R a+x ~/.venv
+    # activate
+    . ~/.venv/pycraft/bin/activate # on mac
+    . ~/.venv/pycraft/Scripts/activate # on windows
+    # deactivate (when you're done)
+    deactivate
+
+Installing
+----------
+
+.. code:: bash
+
+    pip install -e .
+
+**option 1:**
+
+.. code:: bash
+
+    pip install -e .[dev]
+    # or: python3 setup.py develop
```

Comment:
I just mentioned that setuptools-scm manages that just fine.

The problem with manually doing it is:

 * Like all manual tasks that can be automated, it's error prone
 * Non-stable (eg: from git) builds don't have a clear version to identify them
 * It pollutes the git history with changes that merely push the version - which is redundant when you have tags.

This does not conflict with using a master/dev branch during development. This is about versioning, not workflows.

AFAIK, there are no clear downsides to using it either.